src/Dfc.CourseDirectory.Web/Startup.cs

```diff
@@ -82,6 +83,7 @@ public void ConfigureServices(IServiceCollection services)
             });
 
             services.AddSingleton<IConfiguration>(Configuration);
+            services.AddTransient<IUkrlpWcfClientFactory, UkrlpWcfClientFactory>();
```

Comment:
Singleton?

Reply:
I tend to use `Singleton` where I can, even in trivial places. (Lots of trivial performance hits = large cumulative performance hit)